content/2020/2020-07/2020-07-10-pulsar-weekly.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "2020-07-10-pulsar-weekly"
+displayTitle: "Pulsar Community Weekly Update - 2020-07-04 ~ 2020-07-10"
+date: 2020-07-04 ~ 2020-07-10
+description: "This is the Pulsar community weekly update for 2020-07-04 ~ 2020-07-10, with updates on Pulsar authentication, connectors, Functions, geo replication, and so on."
+id: "2020-07-10-pulsar-weekly"
+---
+
+## [Pulsar Community Weekly Update] 2020-07-04 ~ 2020-07-10
+
+This is the Pulsar community weekly update for 2020-07-04 ~ 2020-07-10, with updates on Pulsar authentication, connectors, Functions, geo replication, and so on.
+
+### Notable Feature
+
+- [Broker] Use CGroup CPU usage when present.
+
+    https://github.com/apache/pulsar/pull/7475
+    
+- [C++ Client] Add support for oauth2 auth.
+
+    https://github.com/apache/pulsar/pull/7467
+    
+- [Function] Differentiate authorization between source/sink/function operations.
+
+    https://github.com/apache/pulsar/pull/7466
+    
+- [Function] Allow function rebalance to be run periodically.
+
+    https://github.com/apache/pulsar/pull/7449
+    
+### Notable Bug Fix
+
+- [Gep Replication] Handle `NoSuchElemetnException` when update geo replication policy.
+
+    https://github.com/apache/pulsar/pull/7514
+    
+- [Function] Fixed deadlock between create function and leader initialization.
+
+    https://github.com/apache/pulsar/pull/7508
+    
+- [Broker] Fixed get last entry is trying to read entry -1.
+
+    https://github.com/apache/pulsar/pull/7495
+    
+- [Broker] Fix `ArrayIndexOutOfBoundsException` when dispatch messages to consumer.
+
+    https://github.com/apache/pulsar/pull/7483
+    
+- [Broker] Avoid the NPE occurs in method `ManagedLedgerImpl.isOffloadedNeedsDelete`.
+
+    https://github.com/apache/pulsar/pull/7389
+    
+- [Broker] Fix update partitions error for non-persistent topic.
+
+    https://github.com/apache/pulsar/pull/7459
+    
+- [Java Client] Fix batch ackset recycled multiple times.
+
+    https://github.com/apache/pulsar/pull/7409
+
+### Event / News
+
+- [Pulsar Summit] The first-ever Pulsar Summit Virtual Conference 2020 was held on June 17 - 18. This two-day digital event featured more than 30 sessions from top Pulsar contributors and developers.
```

Comment:
shall we add this again?

Reply:
Yes, like we promoting Summit on Twitter this week.